src/types/account-memberships.d.ts

```diff
@@ -42,6 +42,20 @@ export interface AccountMembershipsIncluded {
 type AccountMembershipsInclude = | 'account_members'
 
 type AccountMembershipsResponse = ResourcePage<AccountMembership, AccountMembershipsIncluded>
+
+
+/**
+ * filter for account memberships
+ */
+export interface AccountMembershipsFilter {
+  eq?: {
+    account_member_id?: string
+  }
+  like?: {
+    account_member_id?: string
```

Comment:
What is the purpose of `like` when filtering ids? Do we actually support filtering on the part of an id?

Reply:
that's the requirement in my ticket. https://elasticpath.atlassian.net/browse/MT-6766

Reply:
I think either @epmkelly or @steve-r-west groomed that, so there must have been a reason

Reply:
I groomed that, and might have gone over all those tickets at the same time, this was the only field. maybe it doesn't make a lot of sense to have partial match on ids? but i guess it doesn't really hurt to have the logic, so i figured to have it anyways.

Reply:
👍